worker/Dockerfile.template

```diff
@@ -38,10 +40,9 @@ RUN mkdir -p /etc/qemu \
 WORKDIR /usr/app
 
 COPY config config
-COPY entry.sh .
 
 COPY --from=node-build /tmp/node/package.json .
 COPY --from=node-build /tmp/node/node_modules node_modules
 COPY --from=node-build /tmp/node/build build
 
-CMD [ "./entry.sh" ]
+CMD ["node", "./build/bin"]
```

Comment:
This would conflict with [your UNIX socket changes](https://github.com/balena-os/leviathan/pull/539) as you are still using the entrypoint script in that PR.

Reply:
Good point, and we don't need to get rid of the entry script, we can just replace the start command inside it.